src/app/cli/src/explore/api_server.rs

```diff
@@ -95,6 +96,7 @@ impl APIServer {
                     )
                     .layer(Extension(base_catalog))
                     .layer(Extension(gql_schema))
+                    .layer(kamu_adapter_http::RunInDatabaseTransactionLayer::new())
```

Comment:
It's probably okay for the start, but I think we would want more specific per-route application. Some HTTP routes likely do not not require database access.

Reply:
Yes, I agree with you -- but it's not yet a finished PR